### PR TITLE
Java8 and refactors

### PR DESCRIPTION
**Pull request by akshaisarma**

### CODE REVIEW

**Comment by akshaisarma** [comment]
Haven't bothered with test code.

**Comment by yahoocla** [comment]
CLA is valid!

File: README.md [hunk 1]
```diff
@@ -81,10 +81,11 @@ Version | Notes
 0.1.7 | Multiple Hive databases across Queries
 0.1.8 | Null types in Hive results fix
 0.1.9 | Empty results handling bug fix
+0.2.0 | Internal switch to Java 8. hive-queue is no longer a setting. Use hive-setting.
 
 ## Members
 
-Akshai Sarma, akshaisarma@gmail.com  
+Akshai Sarma, akshaisarma@gmail.com
```
Comment (joshwalters) [inline README.md]: If you remove the double-space there is no newline.

Comment (akshaisarma) [inline README.md]: Done. Auto stripped trailing spaces.

File: src/main/java/com/yahoo/validatar/execution/EngineManager.java [hunk 2]
```diff
@@ -136,40 +122,37 @@ protected void setEngines(List<Engine> enginesToUse) {
...
+            log.error("Required engine " + engine + " could not be setup.");
+            working.getEngine().printHelp();
+            return false;
+        }
+        return true;
     }
 
     /**
-     * Prints the help message for each engine.
+     * {@inheritDoc}
```
Comment (joshwalters) [inline src/main/java/com/yahoo/validatar/execution/EngineManager.java]: Do we need this or can we just mark with "@override"?

File: src/test/resources/suppressions.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    <suppress checks="Javadoc.*" files="src[\\/]test[\\/]java[\\/].+.java"/>
+</suppressions>
```
Comment (joshwalters) [inline src/test/resources/suppressions.xml]: Missing newline at end of file?

**Comment by joshwalters** [comment]
Looks good! The new java8 streams stuff is quite nice.

File: src/main/java/com/yahoo/validatar/execution/EngineManager.java (hunk 2 above)
Comment (akshaisarma) [inline src/main/java/com/yahoo/validatar/execution/EngineManager.java]: Good call. There were many others. I fixed them.

File: src/test/resources/suppressions.xml (hunk 3 above)
Comment (akshaisarma) [inline src/test/resources/suppressions.xml]: Done.

**Pull request merged**